config/locales/idv/en.yml

```diff
@@ -90,10 +90,19 @@ en:
         accurate. To learn more about the information we need from you and how it’s used
         to verify your identity, please see our troubleshooting page.
       finance:
-        label: Select which type of financial information you would like to provide.
-        intro: >
-          You are not sharing account balances or transaction details, and we
-          will never charge any fees.
+        hint: "You can use Mastercard, Visa, Diner's Club, or JCB."
+        intro_ccn: >
+          Help us verify your identity by providing the last 8 digits of one of your
+          credit cards.
+        intro_account: >
+          Help us verify your identity by providing an auto loan, mortgage, or home equity loan
+          number.
+        no_ccn: Don't have a credit card handy?
+        no_ccn_info_html: You can also veridy with an auto loan, mortgage, or home equity loan
```

Comment:
typo: `s/veridy/verify/g`

Also, if the HTML is just at the end like this, should we just not make these HTML keys and put the link in next to the text in the template?